.changeset/config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@1.5.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "linked": [],
+  "access": "public",
+  "baseBranch": "master",
+  "updateInternalDependencies": "patch",
+  "ignore": []
```

Comment:
I guess this configuration would also publish the **demo** package, which we don't want.

The official answer from [changeset](https://github.com/atlassian/changesets/blob/master/docs/config-file-options.md#ignore-array-of-packages) would be to set `private: true` under `demo/package.json` but for some reason the private key is removed every time I run `yarn`.

So I'm not sure if there's a fix, I'll try to investigate. If there's no fix, we'll have to add `demo` to the `ignore` array.

Reply:
yeah I think the ignore array makes most sense for this: https://github.com/n1ru4l/graphql-live-query/blob/d14f5fcdbf7a595477c074e32b5702dc306d1441/.changeset/config.json#L11

Reply:
Did you try setting `private:true` in your repo? It seems like this would be the recommended option, but yarn workspaces breaks it for some reason. Anyway, let's move with the ignore array for now.

Reply:
Afaik private is to indicate that a package should be published as a private npm package